repository/src/main/java/org/apache/atlas/repository/audit/CassandraBasedAuditRepository.java

```diff
@@ -217,7 +253,22 @@ void createSession() throws AtlasException {
     Cluster.Builder cassandraClusterBuilder = Cluster.builder();
 
     String hostname = APPLICATION_PROPERTIES.getString(CASSANDRA_HOSTNAME_PROPERTY, "localhost");
-    Cluster cluster = cassandraClusterBuilder.addContactPoint(hostname).withClusterName(clusterName).withPort(port).build();
+
+    Cluster cluster = null;
+    if (isSSLEnabled) {
+      if (StringUtils.isNotEmpty(username) && StringUtils.isNotEmpty(password)) {
```

Comment:
User credentials should not be locked into ssl, there are scenarios where only authentication is enought.